app/controllers/api/base_controller/manager.rb

```diff
@@ -140,7 +140,7 @@ def update_multiple_collections(is_subcollection, target, type, resources)
           update_one_collection(is_subcollection, target, type, rid, r)
         end
         raise BadRequestError, "No #{type} resources were specified for the #{action} action" if processed == 0
-        {"results" => results}
+        {"results" => results.flatten}
```

Comment:
@imtayadeway it wasn't failing - I believe this is the first action where it is returning an array of results per resource, so the results were being returned in a nested array (`[[]]`) rather than how they are now, but `.flatten` fixes that. Does that make sense?